dataprofiler/data_readers/data_utils.py

```diff
@@ -232,7 +261,8 @@ def read_json(data_generator, selected_columns=None, read_in_string=False):
                 ),
                 ignore_dicts=True,
             )
-            lines.append(obj)
+            if isinstance(obj, dict):  # should always pass but needed for mypy
+                lines.append(obj)
```

Comment:
based on unicode_to_str could be str, list, or dict?

Reply:
that's what it looks like in the func. The input can also be a list, dict, or str. 

basically converts ensures any str inside of the objs are str and not unicode

Reply:
@Sanketh7 why just a `dict` here?

Reply:
Good catch and Q @taylorfturner